examples/use-kits.yaml

```diff
@@ -0,0 +1,116 @@
+    '@type': RecipeProcess
+    skos:note: Hold a workshop with meal
+    time:hasDuration:
+      time:unitType: time:unitHour
+      time:numericDuration: 4
+
+  - '@id': ouikit:ff208521-9fec-4bb4-a6d3-197f6735cb26
+    '@type': RecipeFlow
+    action: use
+    resourceConformsTo: ouikit:a7e94dde-2997-451e-a7e1-8d364d5e6dee  # reusable eating kit
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+    time:hasDuration:
+      time:unitType: time:unitHour
+      timel:numericDuration: 1
+    reciprocityQuantity:
+      qudt:unit: unit:PERCENT
+      qudt:numericValue: 0.1
+    scalingFactor:
+      - factorProperty: involvesQuantity
+        factorVariable: V
+      - factorProperty: reciprocityQuantity
+        factorVariable: R
+        factorForumula: V*R
+
+  # TODO: add output which scales based on Factors
+
+  # [..... many other inputs and outputs....]
+
+  # a resource
+
+  - '@id': ouikit:9396d2d1-ded3-4b1c-8144-1c024e641039
+    '@type': EconomicResource
+    resourceConformsTo: ouikit:a7e94dde-2997-451e-a7e1-8d364d5e6dee  # reusable eating kit
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+
+  # planning the event
+
+  - '@id': ouikit:a3277840-54b9-497e-9cf6-1902eda1584e
+    '@type': Process
+    name: Workshop dinner 2019-03-15
+    time:hasDuration:
+      time:unitType: time:unitHour
+      time:numericDuration: 4
+
+  - '@id': ouikit:bd5806d6-4a36-45b9-b3b6-3e7d361a5a13
+    '@type': Commitment
+    action: use
+    provider: http://www.ouikit.org/
+    receiver: http://www.ouikit.org/
+    involves: ouikit:9396d2d1-ded3-4b1c-8144-1c024e641039 # ouikits resource
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 30
+    time:hasDuration:
+      time:unitType: time:unitHour
+      time:numericDuration: 4
+    reciprocityQuantity:
+      qudt:unit: unit:PERCENT
+      qudt:numericValue: 3
+    hasBeginning: 2019-03-15T13:00:00-5:00
+    hasEnd: 2019-03-15T17:00:00-5:00
+
+  # [..... many other inputs and outputs....]
+
+  # using the kits for the event
+
+  - '@id': ouikit:30dad10d-606e-4188-80a2-4a1cf2454f58
+    '@type': EconomicEvent
+    action: use
+    provider: http://www.ouikit.org/
+    receiver: http://www.ouikit.org/
+    involves: ouikit:9396d2d1-ded3-4b1c-8144-1c024e641039 # ouikits resource
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 29
+    time:hasDuration:
+      time:unitType: time:unitHour
+      time:numericDuration: 4
+    reciprocityQuantity:
+      qudt:unit: unit:PERCENT
+      qudt:numericValue: 3
+    hasBeginning: 2019-03-15T14:30:00-5:00
+    hasEnd: 2019-03-15T17:00:00-5:00
+
+  - '@id': ouikit:cc5a46e2-b903-4ff5-8c65-587272aba071
+    '@type': Fulfillment
+    fulfills: ouikit:bd5806d6-4a36-45b9-b3b6-3e7d361a5a13
+    fulfilledby: ouikit:30dad10d-606e-4188-80a2-4a1cf2454f58
+    fulfillmentCovers:
+      - coverageProperty: involvesQuantity
+        coverageQuantity: 20
```

Comment:
Is this 20 significant as opposed to 29?  Or typo?

Reply:
just typo